wp-cli/class-vip-user-cli-command.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace Automattic\VIP\CLI;
+
+use Automattic\VIP\Helpers\User_Cleanup;
+
+class VIP_User_CLI_Command extends \WPCOM_VIP_CLI_Command {
+
+	/**
+	 * Remove access for a user from an environment.
+	 *
+	 * Allows lookup via email.
+	 *
+	 * ## OPTIONS
+	 *
+	 * [--emails=<emails>]
+	 * : comma-separated list of emails
+	 */
+	public function cleanup( $args, $assoc_args ) {
```

Comment:
I think a confirmation prompt would be nice here, just in case.

Reply:
This has been added.